src/rendering/batcher/shared/BatchTextureArray.ts

```diff
@@ -29,7 +29,7 @@ export class BatchTextureArray
             const t = this.textures[i];
 
             this.textures[i] = null;
-            this.ids[t.uid] = null;
```

Comment:
you are not wrong, but delete is a suuppperr slow operation

Reply:
Yeah, but I imagine if these object have tens of thousands of unnecessary entries that can't be good either. In theory you could probably run out of memory.

Reply:
maybe there is like a middle ground - could just reset them every X frames ?

Reply:
My main concern is that temoving keys changes the object's shape and can lead to de-optimization.